.circleci/config.yml

```diff
@@ -1244,12 +1269,31 @@ jobs:
                   test/externalTests/<<parameters.project>>.sh solcjs /tmp/workspace/soljson.js
       - when:
           condition:
-            equal: [<< parameters.binary_type >>, "native"]
+            and:
+                - equal: [<< parameters.binary_type >>, "native"]
+                - equal: [<< parameters.script_lang >>, "shellscript"]
           steps:
             - run:
                 name: External <<parameters.project>> tests (native)
                 command: |
                   test/externalTests/<<parameters.project>>.sh native /tmp/workspace/solc/solc
+      - when:
+          condition:
+            and:
+                - equal: [<< parameters.binary_type >>, "native"]
+                - equal: [<< parameters.script_lang >>, "python3"]
+          steps:
+            - run:
+                name: Install script dependencies
+                command: |
+                    sudo apt update
+                    sudo apt install python3-pip --assume-yes --no-install-recommends
+                    python3 -m pip install requests --user
```

Comment:
We do this in more than one place. We should add a reusable step for installing Python.